src/googleclouddebugger/firebase_client.py

```diff
@@ -410,7 +427,7 @@ def _TransmitBreakpointUpdates(self):
       try:
         # Something has changed on the breakpoint.
         # It should be going from active to final, but let's make sure.
-        if not breakpoint_data['isFinalState']:
+        if not breakpoint_data.get('isFinalState'):
```

Comment:
If you add False for the default argument, possibly makes it more readable?

Reply:
sgtm